kubernetes/helm/presto/templates/coordinator/configmap.yaml

```diff
@@ -30,6 +30,7 @@ data:
     pinot.use-date-trunc={{ .Values.pinot.useDateTrunc }}
     pinot.infer-date-type-in-schema={{ .Values.pinot.inferDateTypeInSchema }}
     pinot.infer-timestamp-type-in-schema={{ .Values.pinot.inferTimestampTypeInSchema }}
+    pinot.use-streaming-for-segment-queries={{ .Values.pinot.useStreamingForSegmentQueries }}
```

Comment:
You can remove entire `pinot.properties`
then add:
```
{{- range $path, $config := .Values.extraConfigs }}
  {{ $path }}: |
{{- tpl $config $ | nindent 4 -}}
{{- end -}}
```

Reply:
Done